datakyt/_config.py

```diff
@@ -0,0 +1,13 @@
+_global_config = {'db_name': 'datakyt',
+                  'db_admin': 'datakyt_admin',
+                  'db_password': ''}
+
+
+def get_config() -> dict:
```

Comment:
Add documentation

Reply:
I've added documentation, please check the updates